packages/desktop-client/src/components/filters/FilterExpression.tsx

```diff
@@ -62,8 +62,21 @@ export function FilterExpression<T extends RuleConditionEntity>({
         variant="bare"
         isDisabled={customName != null}
         onPress={() => setEditing(true)}
+        style={{
```

Comment:
<img width="352" alt="Screenshot 2024-10-11 at 19 49 20" src="https://github.com/user-attachments/assets/d44046db-0801-4912-85bf-62edce06d933">

fix for filters expanding out-of-bounds